src/modules/party-external-rating/party-external-rating.service.test.ts

```diff
@@ -1,33 +1,33 @@
-import { AcbsService } from '@ukef/modules/acbs/acbs.service';
+import { AcbsPartyExternalRatingService } from '@ukef/modules/acbs/acbs-party-external-rating.service';
 import { PartyExternalRatingGenerator } from '@ukef-test/support/generator/party-external-rating-generator';
 import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
 import { when } from 'jest-when';
 
 import { AcbsAuthenticationService } from '../acbs/acbs-authentication.service';
 import { PartyExternalRatingService } from './party-external-rating.service';
 
-jest.mock('@ukef/modules/acbs/acbs.service');
+jest.mock('@ukef/modules/acbs/acbs-party-external-rating.service');
 jest.mock('@ukef/modules/acbs/acbs-authentication.service');
 
-describe('AcbsPartyExternalRatingService', () => {
+describe('PartyExternalRatingService', () => {
   const valueGenerator = new RandomValueGenerator();
   const authToken = valueGenerator.string();
 
   let acbsAuthenticationService: AcbsAuthenticationService;
-  let acbsService: AcbsService;
+  let acbsService: AcbsPartyExternalRatingService;
   let service: PartyExternalRatingService;
 
   beforeEach(() => {
     acbsAuthenticationService = new AcbsAuthenticationService(null, null, null);
-    acbsService = new AcbsService(null, null);
+    acbsService = new AcbsPartyExternalRatingService(null, null);
     service = new PartyExternalRatingService(acbsAuthenticationService, acbsService);
 
     // eslint-disable-next-line jest/unbound-method
```

Comment:
Appreciate this is not in the PR but can we please remove inline eslint rule and resolve the linting issue. Thank you.